apps/api/v2/src/modules/organizations/repositories/organizations-teams.repository.ts

```diff
@@ -49,6 +49,22 @@ export class OrganizationsTeamsRepository {
     });
   }
 
+  async findOrgUserTeamsPaginated(organizationId: number, userId: number, skip: number, take: number) {
+    return this.dbRead.prisma.team.findMany({
+      where: {
+        parentId: organizationId,
+        members: {
+          some: {
+            userId,
+            accepted: true,
```

Comment:
I think we can also include not accepted teams as long as we include accepted in the response.

Reply:
sounds good